geomstats/geometry/product_manifold.py

```diff
@@ -87,47 +93,57 @@ def regularize(self, point, point_type=None):
 
         Parameters
         ----------
-        point : array-like
+        point : array-like, shape=[n_samples, dim]
+                           or shape=[n_samples, dim_2, dim_2]
+            Point.
         point_type : str, {'vector', 'matrix'}
+            Representation of point.
 
         Returns
         -------
-        regularize_points : array-like
+        regularized_point : array-like, shape=[n_samples, dim]
+                           or shape=[n_samples, dim_2, dim_2]
```

Comment:
Should this be indented to the same height of "array-like"? Not sure what indentation convention is in docstrings.

Reply:
Not sure either, I have indented it as you recommended.